browser/importer/brave_profile_writer.cc

```diff
@@ -1,504 +0,0 @@
-/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
cc: @simonhong: please confirm that these aren't needed any more.

Reply:
Yes, it's just leftover.